examples_utils/benchmarks/environment_utils.py

```diff
@@ -26,6 +45,14 @@
                           "running."),
 }
 
+FALLBACK_VAR_FUNCTIONS: Dict[str, Tuple[Optional[Union[Callable[[], Optional[str]], str]], ...]] = {
```

Comment:
Do we need to type hint here? Feels its more confusing than helpful here

Reply:
It's not needed but if you turn on (light) type checking  in VS Code's pylance options it correctly figures out the types of your variables and puts squiggles when you make a mistake/forget a branch.

Do you want me to remove it?

Reply:
Ok removed.